### PR TITLE
`General`: Improve user interface of scores page

### PR DESCRIPTION
**Pull request by JohannesStoehr**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/guidelines/development-process/#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context

Fix some small things related to the ui of the scores table.

### Description

Shorten the headlines of student/team names (to just "Names"), shorten the headline of the indicator for the practice mode (to just "practice").
Fix a clipping error when exporting submissions/repos for smaller screens (see screenshot). Now the first popover closes when the second larger for the actual export is opened.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Programming Exercise

1. Log in to Artemis
2. Navigate to the scores table of the exercise
3. Verify these small fixes

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### Screenshots

Wrong clipping (the export options should be in the back or even closed again):
image
New table headings:
Bildschirm­foto 2023-03-25 um 09 08 48

### CODE REVIEW

File: src/main/webapp/i18n/de/participation.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
             "student": "Student",
             "team": "Team",
             "students": "Studierende",
-            "practiceMode": "Übungsmodus",
+            "practice": "Praxis",
```
Comment (pal03377) [inline src/main/webapp/i18n/de/participation.json:44]: I don't think this translation is a fitting change from "Übungsmodus" for "practice" 🤔  Maybe "Üben"?

**Review by pal03377 — APPROVED** [review]
Tested on TS1, also the code looks good apart from one minor translation issue

**Review by terlan98 — APPROVED** [review]
Tested on ts2. Works as expected. The code also looks good.

File: src/main/webapp/i18n/de/participation.json (hunk 1 above)
Comment (simon-hng) [reply src/main/webapp/i18n/de/participation.json:44]: I prefer "üben" to Praxis

**Review by simon-hng — DISMISSED** [review]
Code changes look good. One small translation issue

File: src/main/webapp/i18n/de/participation.json (hunk 1 above)
Comment (krusche) [reply src/main/webapp/i18n/de/participation.json:44]: I agree that Praxis does not fit. The German translation is a bit difficult here. We should probably keep "Übungsmodus" here

**Review by terlan98 — APPROVED** [review]
Reapproved

**Pull request merged**